primeNbr.js

```diff
@@ -0,0 +1,14 @@
+const isPrime = (number) => {
+    if (number < 1 ) return false
+
+    let isPrimeNumber = true
+    for (let index = 2; index<number; index++){
+        if (number % index === 0){
+            isPrimeNumber = false
+            break
```

Comment:
break is not necessary in this case, you can implement early return in this case, ```return false``` can help you in such situation